sdk/communication/communication-common/src/autoRefreshTokenCredential.ts

```diff
@@ -24,6 +24,13 @@ export interface CommunicationTokenRefreshOptions {
    * By default false.
    */
   refreshProactively?: boolean;
+
+  /**
+   * The time span before token expiry that causes the 'tokenRefresher' to be called if 'refreshProactively' is true.
+   * For example, setting it to a value equal to 5 minutes means that 5 minutes before the cached token expires, the proactive refresh will request a new token.
+   * By default, the value is equal to 10 minutes.
+   */
+  refreshTimeBeforeTokenExpiryInMs?: number;
```

Comment:
Why ms? Seconds or minutes seems more reasonable unit to me.

Reply:
I switched it to sec